dpbench/configs/bench_info/rambo.toml

```diff
@@ -42,9 +42,12 @@ nout = 4
 
 [benchmark.init]
 func_name = "initialize"
+types_dict="types_dict"
```

Comment:
Is there any better way to name it? It's kind of confusing, that the type itself is dictionary, while it is string.

Reply:
Done